Vertexvis/vertex-web-sdk
# [PLAT-2148] Volume intersection query tool

## Summary

Adds a new `<vertex-viewer-box-query-tool>` element that supports dragging to draw boxes over portions of the viewer and performing various operations on the contents of that specified volume (this defaults to selection-based operations, but the `controller` supports replacing with any operation).

## Test Plan

- Add a `<vertex-viewer-box-query-tool>` underneath the viewer and verify that drag selection works
- Set the `defaultOperation` to `deselect` and verify that drag deselection works
- Set a custom operation on the `controller` using `setOperationTransform` and verify that operation works with the query
- Add and style a custom drag overlay

## Release Notes

N/A

## Possible Regressions

N/A - new functionality

## Dependencies

N/A
